cli/client/opts.go

```diff
@@ -129,5 +137,41 @@ func NewOptsFromEnv() (*GohanClientCLIOpts, error) {
 	}
 	opts.gohanSchemaURL = gohanSchemaURL
 
+	outputFormatOpt := os.Getenv(outputFormatEnvKey)
+	if outputFormatOpt != "" {
+		outputFormat, err := findOutputFormat(outputFormatOpt)
+		if err != nil {
+			return nil, err
+		}
+		opts.outputFormat = outputFormat
+	}
+
+	verbosity := os.Getenv(logLevelEnvKey)
+	if verbosity != "" {
+		logLevel, err := parseLogLevel(verbosity)
+		if err != nil {
+			return nil, err
+		}
+		opts.logLevel = logLevel
+	}
+
 	return &opts, nil
 }
+
+func findOutputFormat(formatOpt interface{}) (string, error) {
+	for _, format := range outputFormats {
+		if format == formatOpt {
+			return format, nil
+		}
+	}
+	return "", fmt.Errorf(incorrectOutputFormat, outputFormats)
+}
+
+func parseLogLevel(verbosityOpt interface{}) (logging.Level, error) {
+	for i, logLevel := range logLevels {
+		if fmt.Sprint(i) == verbosityOpt {
+			return logLevel, nil
+		}
+	}
+	return 0, fmt.Errorf(incorrectVerbosityLevel, 0, len(logLevels)-1)
```

Comment:
I'll change this, but it shouldn't matter, since callers shouldn't use the return value in case of an error. But if they should be able to, shouldn't `findOutputFormat()` return a default value (e.g. "table") as well?